<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>556</height>
        </rect>
       </property>
       <widget class="QTextEdit" name="NuPro">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>151</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Number of processes:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="StartBut">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>260</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Empty Holes!</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>320</y>
          <width>801</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ShowProcesses">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>220</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
    color: silver;
    background-color: #302F2F;
    border-width: 1px;
    border-color: #4A4949;
    border-style: solid;
    padding-top: 5px;
    padding-bottom: 5px;
    padding-left: 5px;
    padding-right: 5px;
    border-radius: 2px;
    outline: none;
}

QPushButton:disabled
{
    background-color: #302F2F;
    border-width: 1px;
    border-color: #3A3939;
    border-style: solid;
    padding-top: 5px;
    padding-bottom: 5px;
    padding-left: 10px;
    padding-right: 10px;
    /*border-radius: 2px;*/
    color: #454545;
}

QPushButton:focus {
    background-color: #00FF00;
    color: black;
}
QPushButton:checked{
    background-color: #4A4949;
    border-color: #6A6969;
}
QPushButton:hover
{
    border: 1px solid #00FF00;
    color: #00FF00;
	background-color: #4A4949;
}</string>
        </property>
        <property name="text">
         <string>Run!</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>111</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox { 
     border: 2px solid white; 
     border-radius: 3px; 
 } </string>
        </property>
        <property name="title">
         <string>Choose</string>
        </property>
        <widget class="QRadioButton" name="FirstFit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>117</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>First Fit</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="BestFit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>117</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Best Fit</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="WorstFit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>70</y>
           <width>117</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Worst Fit</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>171</width>
          <height>171</height>
         </rect>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox { 
     border: 2px solid white; 
     border-radius: 3px; 
 } </string>
        </property>
        <property name="title">
         <string>Holes</string>
        </property>
        <widget class="QTextEdit" name="HoSize">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>71</width>
           <height>111</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="StartAdd">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>50</y>
           <width>71</width>
           <height>111</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>71</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Sizes (K):</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Adresses:</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>10</y>
          <width>171</width>
          <height>171</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox { 
     border: 2px solid white; 
     border-radius: 3px; 
 } </string>
        </property>
        <property name="title">
         <string>Processes</string>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>71</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Sizes (K):</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="Processes">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>49</y>
           <width>71</width>
           <height>111</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="ProSize">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>50</y>
           <width>71</width>
           <height>111</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>51</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Names:</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>230</y>
          <width>291</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox { 
     border: 2px solid white; 
     border-radius: 3px; 
 } </string>
        </property>
        <property name="title">
         <string>Deallocation</string>
        </property>
        <widget class="QTextEdit" name="DeletedP">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>30</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="DeAloocat">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>97</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Deallocate</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>0</y>
          <width>191</width>
          <height>201</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox { 
     border: 2px solid white; 
     border-radius: 3px; 
 } </string>
        </property>
        <property name="title">
         <string>States</string>
        </property>
        <widget class="QTextEdit" name="State">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>171</width>
           <height>161</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>190</y>
          <width>281</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox { 
     border: 2px solid white; 
     border-radius: 3px; 
 } </string>
        </property>
        <property name="title">
         <string>Insert New Process</string>
        </property>
        <widget class="QTextEdit" name="InsertedName">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>40</y>
           <width>104</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="InsertedSize">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>80</y>
           <width>104</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="Insert">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>60</y>
           <width>97</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>Insert</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>51</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>41</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
